contributing/content-style.md

```diff
@@ -281,7 +281,7 @@ To login, run `platform login`.                               | To login, run `p
 
 Notes are pieces of information that stand outside the normal text flow.
 Use them for short ideas that aren't.
-Don't use too many in one place or they lose their value.
+Don't use too many notes in one place or they lose their value.
```

Comment:
:hammer_and_wrench: I like your focus above on taking out unnecessary words. I feel like `notes` here isn't necessary, so I'd keep that trend and remove it here.
 ```suggestion
Don't use too many in one place or they lose their value.
```